README.md

```diff
@@ -85,7 +85,7 @@ Currently the codebase supports the following runtime environments:
 
 1. Node.js 20+
 2. Modern browsers (Chromium/Firefox/Safari, no Internet Explorer or
-   [Edge Spartan](https://en.wikipedia.org/wiki/Microsoft_Edge#Development))
+   [Edge Spartan](https://en.wikipedia.org/wiki/Microsoft_Edge_Legacy#Development))
```

Comment:
```suggestion
   [Edge Legacy](https://en.wikipedia.org/wiki/Microsoft_Edge_Legacy))
```

If this has its own wikipedia article now we can refer to the top title instead of a barely known development codename